nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/helpers/AbstractTestColdStorageHelper.java

```diff
@@ -41,30 +34,25 @@
 import org.apache.commons.collections.CollectionUtils;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.nuxeo.coldstorage.blob.providers.DummyBlobProvider;
 import org.nuxeo.ecm.core.api.*;
 import org.nuxeo.ecm.core.api.security.ACE;
 import org.nuxeo.ecm.core.api.security.ACL;
 import org.nuxeo.ecm.core.api.security.ACP;
 import org.nuxeo.ecm.core.api.security.SecurityConstants;
 import org.nuxeo.ecm.core.blob.BlobManager;
-import org.nuxeo.ecm.core.blob.BlobStatus;
 import org.nuxeo.ecm.core.blob.ManagedBlob;
 import org.nuxeo.ecm.core.event.impl.DocumentEventContext;
 import org.nuxeo.ecm.core.event.test.CapturingEventListener;
 import org.nuxeo.ecm.core.io.download.DownloadService;
-import org.nuxeo.coldstorage.ColdStorageFeature;
 import org.nuxeo.ecm.core.test.CoreFeature;
-import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
 import org.nuxeo.runtime.test.runner.TransactionalFeature;
 
 /**
  * @since 11.1
  */
 @RunWith(FeaturesRunner.class)
-@Features(ColdStorageFeature.class)
```

Comment:
I am wondering if you can create a new feature like you did for the S3. The new one will be called `DefaultColdStorageFeature` it will be configured with the `dummy` configuration and be use. Later we can even use them in the operation suit tests to check both of S3 and default/dummy